data.sql

```diff
@@ -42,4 +42,39 @@ UPDATE animals set owner_id = 3 WHERE name in ('Devimon','Plantmon');
 UPDATE animals set owner_id = 4 WHERE name in ('Charmander','Squirtle','Blossom');
 
 UPDATE animals set owner_id = 5 WHERE name in ('Angemon','Boarmon');
-       
\ No newline at end of file
+       
+--        Day 4
+
+INSERT INTO vets(id,name, age, date_of_graduation)
+	VALUES 	(1,'William Tatcher', 45, '2000-4-23'),
+			(2,'Maisy Smith', 26, '2019-2-17'),
+			(3,'Stephanie Mendez', 64, '1981-5-4'),
+			(4,'Jack Harkness', 38, '2008-1-8');
+
+INSERT INTO specializations (id,vet_id, species_id)
+	VALUES 	(1,1,1),
+			(2,3,1),
+			(3,3,2),
+			(4,4,2);
```

Comment:
- [ ] The same thing with the specializations table. The id will be generated automatically, so there's  no need to add it manually:
```sql
INSERT INTO specializations (vet_id, species_id)
	VALUES 	(1,1);
```